src/main/java/race/Cars.java

```diff
@@ -9,9 +10,11 @@
     private final List<Car> cars = new ArrayList<>();
 
     private Cars(Condition condition) {
-        for (int i = 0; i < condition.getCarCount(); i++) {
-            cars.add(Car.of(condition.getTryCount()));
-        }
+        cars.addAll(
+                Arrays.stream(condition.getNames())
+                        .map(name -> Car.of(condition.getTryCount(), name))
+                        .collect(Collectors.toList())
+        );
```

Comment:
```suggestion
        this.cars = Arrays.stream(condition.getNames())
                        .map(name -> Car.of(condition.getTryCount(), name))
                        .collect(Collectors.toList());
```
위와 같이 초기화하는 건 어떨까요~?